src/index.js

```diff
@@ -1,6 +1,7 @@
 var parser = require('graphql/language/parser');
-
+var printer = require('graphql/language/printer');
```

Comment:
While this probably solves the problem thanks the the providing of the much more complete and considerate `printer` module from `graphql-js`, introducing this would be adding the full weight of that printer to client bundles, no?

I'm not sure if that was the original reason for using the more simple implementation, but have you evaluated how this change affects bundle size?

Reply:
I haven't. Unfortunately haven't worked on any GraphQL projects in a while so haven't needed this. 
I'll try to find some time over the next week to get back to this

Reply:
Our team could look at fixing this up so the printer doesn't get included in the client bundle. However, we've never worked on this code base, so have no idea where to start. Can someone provide guidance on an alternative approach that would be accepted? Is there another way we can help get this closer to being merged?